### PR TITLE
Update overflow menu, add field value and disabled property to radio button

### PR DESCRIPTION
**Pull request by aaroncouture**

### CODE REVIEW

File: src/components/Checkbox/Checkbox.jsx [hunk 1]
```diff
@@ -21,26 +17,22 @@ class Checkbox extends React.Component {
       disabled,
       value,
       data,
+      onBlur
```
Comment (aaroncouture) [inline src/components/Checkbox/Checkbox.jsx]: just added onBlur, the rest is just prettier making life difficult

File: src/components/OverflowMenu/OverflowMenu.jsx [hunk 2]
```diff
@@ -1,45 +1,50 @@
...
+  };
 
-  handleClick = (event) => {
+  handleClick = event => {
     this.setState({ revealed: false });
     this.props.onClick(event);
-  }
+  };
+
+  renderLabel = () => {
```
Comment (aaroncouture) [inline src/components/OverflowMenu/OverflowMenu.jsx]: added this renderLabel method

File: src/components/RadioButton/RadioButton.jsx [hunk 3]
```diff
@@ -1,51 +1,55 @@
...
+            className={cx(style.original, {
+              [style.disabled]: this.props.disabled
+            })}
             type="radio"
             id={id}
             name={this.props.name}
             onChange={this.props.onChange}
-            checked={this.props.checked}
+            checked={
+              this.props.checked || this.props.value === this.props.fieldValue
```
Comment (aaroncouture) [inline src/components/RadioButton/RadioButton.jsx]: added support for sending checked or a string value

File: src/components/RadioButton/RadioButton.jsx [hunk 4]
```diff
@@ -1,51 +1,55 @@
...
+            }
+            value={this.props.fieldValue}
```
Comment (aaroncouture) [inline src/components/RadioButton/RadioButton.jsx]: added "fieldValue" prop so we can give the radiobutton a group in a radio group scenario

File: src/components/RadioButton/RadioButton.jsx [hunk 5]
```diff
@@ -1,51 +1,55 @@
...
+            disabled={this.props.disabled}
           />
           <div className={style.faux} />
-          <div className={style.text}>
-            <strong>{ this.props.text }</strong>
+          <div
+            className={cx(style.text, {
+              [style.disabled]: this.props.disabled
```
Comment (aaroncouture) [inline src/components/RadioButton/RadioButton.jsx]: added disabled support here

File: src/components/RadioButton/style.scss [hunk 6]
```diff
@@ -50,16 +60,20 @@
     top: -4px;
   }
 }
-
-.description, .text {
+/*
```
Comment (aaroncouture) [inline src/components/RadioButton/style.scss]: I have to look into why I did this... yikes

File: src/components/Rule/Rule.jsx [hunk 7]
```diff
@@ -1,39 +1,33 @@
...
 
-const type = { SOLID: 'solid', DOTTED: 'dotted', DASHED: 'dashed' };
+const type = { SOLID: "solid", DOTTED: "dotted", DASHED: "dashed" };
 
 class Rule extends React.Component {
   render() {
     const ruleStyle = cx(style.wrapper, {
       [style.dotted]: this.props.type === type.DOTTED,
       [style.dashed]: this.props.type === type.DASHED,
+      [style.withTopMargin]: this.props.withTopMargin
```
Comment (aaroncouture) [inline src/components/Rule/Rule.jsx]: added the ability to specify a top margin (all my mocks have both a top and bottom margin around the Rule

File: src/components/Text/Text.jsx [hunk 8]
```diff
@@ -1,17 +1,16 @@
...
+import { getDataAttrs, getDataProps } from "../../dataUtils";
 
 class Text extends React.Component {
   render() {
+    const { large } = this.props;
     return (
       <div
         {...getDataAttrs(this.props.data)}
-        className={style.wrapper}
+        className={cn(style.wrapper, { [style.large]: large })}
```
Comment (aaroncouture) [inline src/components/Text/Text.jsx]: added a large text option

File: src/components/layout/FormField/FormField.jsx [hunk 9]
```diff
@@ -1,58 +1,49 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import {
-  getDataAttrs,
-  getDataProps,
-} from '../../../dataUtils';
-import style from './style.scss';
-import Label from '../../Label/Label.jsx';
+import React from "react";
```
Comment (aaroncouture) [inline src/components/layout/FormField/FormField.jsx]: no changes in here.need to revert

File: src/components/Card/Card.jsx [hunk 10]
```diff
@@ -80,7 +81,7 @@ class Card extends React.Component {
           { (title || actions) &&
             <div className={style.top}>
               { title &&
-                <Header h3>
+                <Header h4={hasSmallHeader} h3={!hasSmallHeader}>
```
Comment (aaroncouture) [inline src/components/Card/Card.jsx]: added support for a small header.. speaking with Deena, h4 should become the new header size.

File: src/components/Loader/Loader.jsx [hunk 11]
```diff
@@ -1,41 +1,25 @@
...
+          <div className={style.bounce3} style={this.props.dotStyle} />
         </div>
       </div>
     );
   }
 }
 
-Loader.defaultProps = {};
+Loader.defaultProps = {
+  dotStyle: { backgroundColor: '#59698B' },
```
Comment (aaroncouture) [inline src/components/Loader/Loader.jsx]: just added a default background color to the loader.

File: src/components/RadioButton/style.scss (hunk 6 above)
Comment (aaroncouture) [inline src/components/RadioButton/style.scss]: remove this comment

**Review by kangax — APPROVED** [review]

**Review by kangax — APPROVED** [review]
🚢

File: src/components/Card/Card.jsx [hunk 12]
```diff
@@ -70,43 +66,41 @@ class Card extends React.Component {
...
-                <Header h3>
-                  { title }
-                </Header>
-              }
-              { actions &&
-                <div className={style.cardActions}>
-                  { actions.map((action) => action)}
-                </div>
-              }
+              {title && <Header h4>{title}</Header>}
```
Comment (wingyli) [inline src/components/Card/Card.jsx]: wouldn't this affect all existing components using `<Card />`? are we sure we wanna set all headers to `h4`?

**Review by wingyli — APPROVED** [review]

**Comment by kangax** [comment]
@aaroncouture can you plz rebase and update tests :D

File: src/tests/__tests__/__snapshots__/Checkbox.react-test.js.snap [hunk 13]
```diff
@@ -13,6 +13,7 @@ exports[`Checkbox Component render 1`] = `
         disabled={undefined}
         id="id1"
         name="checkbox"
+        onBlur={undefined}
```
Comment (kangax) [inline src/tests/__tests__/__snapshots__/Checkbox.react-test.js.snap]: would be great to test these too but that's minor, don't want to postpone this even further

**Review by kangax — APPROVED** [review]

**Review by wingyli — APPROVED** [review]

**Pull request merged**